javascript/javascript2/week1/movieApp/headerSection.js

```diff
@@ -0,0 +1,39 @@
+const header = document.createElement("header");
+
+/* header */
+header.setAttribute("class", "headerContainer");
+const h1 = document.createElement("h1");
+h1.innerText = "MovieApp";
+header.appendChild(h1);
+
+/* search */
+const searchDiv = document.createElement("div");
+searchDiv.setAttribute("class", "searchDiv");
+
+const searchBox = document.createElement("input");
+searchBox.placeholder = "Search For Movie";
+searchBox.setAttribute("class", "searchInput");
+
+const searchButton = document.createElement("button");
+searchButton.innerText = "Search";
+searchButton.setAttribute("class", "searchButton");
+
+header.appendChild(searchDiv);
+searchDiv.appendChild(searchBox);
+searchDiv.appendChild(searchButton);
+
+searchButton.addEventListener("click", () => {
```

Comment:
You can directly call `handleSearch` like this:

```suggestion
searchButton.addEventListener("click", handleSearch);
```